### ISSUE TITLE
Source detail page - Status tab with list of logs

### ISSUE THREAD

**Issue by johnlafleur** [datalineio/dataline#25]
When clicking on a cell in the list of sources in "Sources", this should display the status detail page"
https://www.figma.com/file/AODY4oi15iI5l0fbxHvobt/Daxtarity?node-id=5782%3A36940

A few features:
- User should be able to click on "Sources" in breadcrumb to go back to the list. 
- List of logs should all be white, but hover state should be grey or red (depending if completed or failed). Hover state should also include the expand / reduce icon on the right of the cells. 
- clicking on any log should expand it: https://www.figma.com/file/AODY4oi15iI5l0fbxHvobt/Daxtarity?node-id=5782%3A36313
- clicking on an expanded log (only the original cell, not the expansion) should reduce it
- user should be able to launch a manual sync with "Sync now" (only the increments)
- in summary cell at the top, only the ENABLED toggle should be clickable. 
- for a manually syncing source, layout should be as follows: https://www.figma.com/file/AODY4oi15iI5l0fbxHvobt/Daxtarity?node-id=5906%3A1233

The v1.0 doesn't include a full refresh. 

@cgardens
